deps.edn

```diff
@@ -4,12 +4,14 @@
         org.clojure/core.rrb-vector {:mvn/version "0.1.1"}
         org.clojure/core.cache      {:mvn/version "1.0.207"}
         com.taoensso/carmine        {:mvn/version "2.20.0"}
-        io.replikativ/konserve      {:git/url "https://github.com/replikativ/konserve.git"
+        io.replikativ/konserve      {:git/url "https://github.com/groundedSAGE/konserve.git"
```

Comment:
Why do we still need your custom konserve version? Can we merge this into konserve beforehand? I would love to synchronize the dependencies bottom up, if this is possible.

Reply:
Sorry I missed this. I have been an advocate for bottom up synchronising for some time. This was due to cascading dependencies on forks from changes required in Incognito that was required for Datahike to compile with advanced compilation.